lib/components/topicButton.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+class TopicButton extends StatelessWidget {
```

Comment:
I am still seeing the file name as:  lib/components/topicButton.dart. It should be  lib/components/topic_button.dart